### PR TITLE
[Home] Component Today Task

### PR DESCRIPTION
**Pull request by irfandigalih**
image

### CODE REVIEW

File: lib/Shared/Widget/TaskList/Component/task_list.dart [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    return Padding(
+      padding: const EdgeInsets.symmetric(vertical: 8),
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.stretch,
+        children: [
+          Row(
+            children: [
+              Expanded(
+                child: Container(
+                  padding: const EdgeInsets.fromLTRB(0, 0, 0, 2),
```
Comment (gilangpersada) [inline lib/Shared/Widget/TaskList/Component/task_list.dart]: pake `EdgeInsets.only()` aja lih

File: lib/Scenes/Page/HomePage/Component/today_task.dart [hunk 2]
```diff
@@ -18,19 +19,54 @@ class _TimeboxingTodayTaskState extends State<TimeboxingTodayTask> {
...
+                const Spacer(),
+                Text('June 23, 2023',
+                    style: TimeBoxingTextStyle.paragraph4(
+                      TimeBoxingFontWeight.bold,
+                      TimeBoxingColors.text20(TimeBoxingColorType.shade),
+                    )),
+              ],
+            ),
+            Padding(
+              padding: const EdgeInsets.fromLTRB(0, 16, 0, 16),
```
Comment (gilangpersada) [inline lib/Scenes/Page/HomePage/Component/today_task.dart]: pake `EdgeInsets.symetric()` aja

File: lib/Shared/Widget/TaskList/Component/task_list.dart (hunk 1 above)
Comment (irfandigalih) [reply lib/Shared/Widget/TaskList/Component/task_list.dart]: done

File: lib/Scenes/Page/HomePage/Component/today_task.dart (hunk 2 above)
Comment (irfandigalih) [reply lib/Scenes/Page/HomePage/Component/today_task.dart]: done

**Review by gilangpersada — APPROVED** [review]

**Review by RenaufalAlrafid — APPROVED** [review]

**Pull request merged**